lib/Immediate/CMakeLists.txt

```diff
@@ -12,9 +12,9 @@ target_link_libraries(swiftImmediate PRIVATE
   swiftIRGen
   swiftSILGen
   swiftSILOptimizer)
-if(LibEdit_FOUND)
+if(LibEdit_HAS_UNICODE)
   target_compile_definitions(swiftImmediate PRIVATE
-    HAVE_LIBEDIT)
+    LIBEDIT_HAS_UNICODE)
```

Comment:
What do you think of rather than changing the internal macro from `HAVE_LIBEDIT` relying on CMake doing the right thing?  That is drop the change here and the changes to REPL.cpp.

Reply:
Sure, works for me.